bitnami/postgresql-ha/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql-ha
-version: 3.8.1
+version: 3.8.2
```

As it's a new feature addition, it's better to bump the minor version 
```suggestion
version: 3.9.0
```